### ISSUE TITLE
Some links on docs.lakefs.io are hard to see

### ISSUE THREAD

**Issue by arielshaqed** [treeverse/lakeFS#6310]
The words "download the binary" [here](https://docs.lakefs.io/howto/deploy/aws.html) ([stable link to v0.104.0](https://docs.lakefs.io/v0.104/deploy/aws.html), other versions may also be affected) are a link:

!["AWS" page section showing "Download the binary to the EC2 instance"](https://github.com/treeverse/lakeFS/assets/7802932/1fcd5326-b769-4be0-bdd5-0ff97e57f037)

They very inaccessible and I am unable to see that this is a link.  I consulted with some well-sighted people, and they _also_ had difficulty distinguishing that this is a link.

**Comment by arielshaqed** [comment]
Marking "critical" as this is on the quickstart for AWS, i.e. one of our most direct drivers on new installations is hard to apply.

**Comment by arielshaqed** [comment]
DOM inspector on Firefox says that this style is `ui-widget-contents`, which is a [jQuery(!?) thing](https://api.jqueryui.com/theming/css-framework/).  All very strange, perhaps we got jQuery from Jekyll?

**Comment by N-o-Z** [comment]
No longer an issue
